model/src/main/java/no/nav/pensjon/vtp/testmodell/brev/BrevMetadataIndeks.kt

```diff
@@ -0,0 +1,22 @@
+package no.nav.pensjon.vtp.testmodell.brev
+
+import com.fasterxml.jackson.core.json.JsonReadFeature
+import com.fasterxml.jackson.databind.json.JsonMapper
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import com.fasterxml.jackson.module.kotlin.readValue
+
+class BrevMetadataIndeks {
+    var brevMetadata : List<BrevMetadata> = emptyList()
+
+    fun init(resource: String) {
+        val mapper = JsonMapper.builder()
+                .addModule(KotlinModule())
+                .configure(JsonReadFeature.ALLOW_BACKSLASH_ESCAPING_ANY_CHARACTER, true)
+                .build()
+        brevMetadata = mapper.readValue(javaClass.getResource(resource).openStream())
+    }
```

Comment:
Syns ikke denne skal lage seg selv. Ta heller å last verdiene i en egen klasse. For eksempel klassen som sender inn `resource` eller BasisdataProviderFileImpl.

Reply:
Enig med deg 😅  Jeg har bare gjort det sånn midlertidig for å flytte denne biten etterpå til BasisdataProviderFileImpl som jeg ikke orket å konvertere til Kotlin akkurat nå.